Add hipFile examples (#465)
amd/rocm-examples: Systems/hipFile/api/get_version/README.md

# hipFile Get Version Example

## Description

> This program demonstrates how to query the hipFile library version at both compile time and runtime.

Reviewer: Same comment as above re. program

> It is the simplest possible hipFile program and a useful first check that the library is installed

Reviewer: I would stay away from saying it's the simplest. 

```suggestion
This simeple hipFile example is a useful first check that the library is installed
```
